src/app/app.component.ts

```diff
@@ -1,10 +1,10 @@
 import { Component } from '@angular/core';
 import { RouterOutlet } from '@angular/router';
-import { HomeComponent } from './home/home.component';
-import { LoginComponent } from './login/login.component';
-import { ProfileComponent } from './profile/profile.component';
-import { RegisterComponent } from './register/register.component';
-import { NavBarComponent } from './components/nav-bar/nav-bar.component';
+import { HomeComponent } from './pages/home/home.component';
+import { LoginComponent } from './pages/login/login.component';
+import { ProfileComponent } from './pages/profile/profile.component';
+import { RegisterComponent } from './pages/register/register.component';
+import { NavBarComponent } from './shared/components/molecules/nav-bar/nav-bar.component';
```

Comment:
No es necesario importar los componentes. Por eso vez que no te funcionan el lazy-loading. te aparecen en el source del browser por que aqui se cargan.

Reply:
I deleted the components except the NavBarComponent bc it's been called on the html 

<img width="727" alt="image" src="https://github.com/lssotog/authentication-app/assets/68559589/71c60a5e-5132-4ad9-9726-067339dad36b">